tests/test_models.py

```diff
@@ -0,0 +1,85 @@
+import pytest
+
+
+@pytest.fixture
+def project():
+    """Create a test project."""
+    from coldfront.core.project.models import Project
+
+    proj = Project()
+    proj.name = "TestProject"
+    return proj
```

Comment:
This is very artificial and that's the reason that the bug in the `__str__` method above was not caught. We already have a `project` test fixture for creating a realistic `Project` instance which should be used instead.